tests/README.md

```diff
@@ -15,6 +17,7 @@ Run the following command to see the current test coverage, including the list o
 that are in need of a test:
 
 ```sh
+# TODO(https://github.com/istio/istio.io/issues/7508): update for the new test framework
```

Comment:
The `make test_status` command is unaffected by this framework change.
```suggestion
```

Reply:
Oh I added this because I saw this [comment](https://docs.google.com/document/d/1r_NoxatNjzPsw0eXr6_9W0rqlkfelt7QfN2y5TF0yTo/edit?disco=AAAAJlQR_W4). I thought you might want to determine whether a doc has a test by checking if there's a test.sh next to snips.sh

Reply:
That will get even more confusing if there are mutliple test.sh files (so the doc might be partially tested). I think we should stick to the rule that if a PR adds a snip.sh file, it also MUST include the corresponding test, at least for now. We can reconsider in the future, if it doesn't work out well.